### PR TITLE
Make test workflow configurable

### PR DESCRIPTION
**Pull request by quesabe**
Closes PLA-202.

### CODE REVIEW

File: src/common/github/pull-request-test-workflow.ts [hunk 1]
```diff
@@ -18,19 +30,52 @@ export class PullRequestTest extends Component {
...
     workflow.addJobs({
       lint: job('npm run lint'),
       typecheck: job('npm run typecheck'),
       test: job('npm run test'),
     })
   }
+
+  /**
+   * Optionally creates a workflow within the given project
+   * or within the project parent (for subprojects).
```
Comment (gvidon) [inline src/common/github/pull-request-test-workflow.ts:54]: @quesabe is the parent project passed through the `project` input param?

**Review by gvidon — CHANGES_REQUESTED** [review]
@quesabe would that be hard to introduce test cases for this functionality? We need to be sure that nested projects can add their workflows to the parent project's `.github/` dir.

**Comment by quesabe** [comment]
> @quesabe would that be hard to introduce test cases for this functionality? We need to be sure that nested projects can add their workflows to the parent project's `.github/` dir.

Not a problem. I should have done it in the first place.

File: src/common/github/pull-request-test-workflow.ts [hunk 2]
```diff
@@ -18,19 +30,52 @@ export class PullRequestTest extends Component {
...
+   *
+   * `hasDefaultGithubWorkflows` option disables workflow creation making the method a simple noop.
```
Comment (gvidon) [inline src/common/github/pull-request-test-workflow.ts:56]: @quesabe not really happy with the noop part and would prefer the method having no such logic, but let’s keep it like this for now.

**Review by gvidon — APPROVED** [review]

**Pull request merged**